spec/index.html

```diff
@@ -805,24 +836,32 @@ <h3>Reified Triples</h3>
     <p>A <a>reified triple</a> is represented using the
       <a href="#grammar-production-reifiedTriple"><code>reifiedTriple</code></a> production
       starting with <a href="#cp-double-lt"><code>&lt;&lt;</code></a>,
-      followed by <a href="#grammar-production-subject"><code>subject</code></a>,
-      <a href="#grammar-production-predicate"><code>predicate</code></a>, and
-      <a href="#grammar-production-ttObject"><code>ttObject</code></a>,
-      optionally follwed by a <a href="#grammar-production-reifier"><code>reifier</code></a>
-      (<a href="#grammar-production-IRIREF"><code>IRIREF</code></a>
-      or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a>),
+      followed by either a <a href="#grammar-production-subject"><code>subject</code></a>
+      or <a href="#grammar-production-reifiedTriple"><code>reifiedTriple</code></a>,
+      a <a href="#grammar-production-predicate"><code>predicate</code></a>, and
+      a <a href="#grammar-production-ttObject"><code>ttObject</code></a>,
+      followed by an optional <a href="#grammar-production-reifier"><code>reifier</code></a>,
+      composed of a <a href="#cp-tilde"><code title="tilde">~</code></a>
+      followed by an optional <a href="#grammar-production-iri"><code>iri</code></a>
+      or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a>,
       and ending with <a href="#cp-double-gt"><code>&gt;&gt;</code></a>.
       For example, `&lt;&lt; :subject :predicate :object ~ :IRIREF &gt;&gt;`.
-      If the optional reifier is not present, a fresh RDF blank node is allocated,
-      as with `&lt;&lt; :subject :predicate :object &gt;&gt;`.</p>
+      If no reifiers are present,
+      or a <a href="#grammar-production-reifier"><code>reifier</code></a>
+      is not immediately followed by an <a href="#grammar-production-iri"><code>iri</code></a>
+      or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a>,
+      a fresh <a data-cite="RDF12-CONCEPTS#dfn-blank-node">RDF blank node</a> is allocated,
+      as with `&lt;&lt; :subject :predicate :object &gt;&gt;`,
+      or `&lt;&lt; :subject :predicate :object ~ &gt;&gt;`.</p>
 
     <p class="note"><a>Reified Triples</a> may be nested,
-      like `&lt;&lt;&nbsp;&nbsp;:subject1&nbsp;:predicate1&nbsp;&lt;&lt;&nbsp;:subject2&nbsp;:predicate2&nbsp;:object2&nbsp;&gt;&gt;&nbsp;~:IRIREF1&nbsp;&gt;&gt;`
-      or `&lt;&lt;&nbsp;:subject4&nbsp;:predicate4&nbsp;&lt;&lt;&nbsp;&nbsp;:subject3&nbsp;:predicate3&nbsp;:object3&nbsp;~:IRIREF3&nbsp;&gt;&gt;&nbsp;&gt;&gt;`.</p>
+      like
+      <br/>`&lt;&lt;&nbsp;:subject1&nbsp;:predicate1 &lt;&lt;&nbsp;:subject2&nbsp;:predicate2&nbsp;:object2&nbsp;&gt;&gt; ~:IRIREF1&nbsp;&gt;&gt;`
+      or <br/>`&lt;&lt;&nbsp;:subject4&nbsp;:predicate4&nbsp;&lt;&lt;:subject3&nbsp;:predicate3&nbsp;:object3&nbsp;~:IRIREF3&nbsp;&gt;&gt;&nbsp;&gt;&gt;`.</p>
```

Comment:
```suggestion
      <br/>`&lt;&lt; :subject1 :predicate1 &lt;&lt; :subject2 :predicate2 :object2 &gt;&gt; ~:IRIREF1 &gt;&gt;`
      or <br/>`&lt;&lt; :subject4 :predicate4 &lt;&lt;:subject3 :predicate3 :object3 ~:IRIREF3 &gt;&gt; &gt;&gt;`.</p>
```

Reply:
It seems this edit suggestion would mess up some entities.

Reply:
I think I fixed it. Look again?

Reply:
BTW, `<br/>` ->  `<br>`. The spec use HTML instead of XHTML.

Reply:
Style choice. It’s legal and worth having such tags self close IMO.